esm/index.js

```diff
@@ -46,23 +39,38 @@ export class Signal {
   /** @returns {T} */
   toJSON() { return this.value }
 
-  /** @returns {T} */
-  toString() { return this.value }
+  /** @returns {string} */
+  toString() { return String(this.value) }
 
   /** @returns {T} */
   valueOf() { return this.value }
 }
 
 let computedSignal;
+/**
+ * @template T
+ * @extends {Signal<T>}
+ */
 class Computed extends Signal {
+  /**
+   * @private
+   * @type{Reactive<T>}
+   */
+  s
+  /**
+   * @param {(v: T) => T} _ 
+   * @param {T} v 
+   * @param {{ equals?: Equals<T> }} o
+   * @param {boolean} f 
+   */
   constructor(_, v, o, f) {
     super(_);
     this.f = f;                   // is effect?
     this.$ = true;                // should update ("value for money")
     this.r = new Set;             // related signals
     this.s = new Reactive(v, o);  // signal
   }
-  /** @readonly */
+  peek() { return this.s.peek() }
```

Comment:
this is inherited ... it shouldn't be needed

Reply:
it was not inherited

Reply:
too many signals libraries ... you are right, I thought Signal extends ReactiveSignal but it's the other way around ... OK then, let's add peek to computed!